jupiterone/data_source_user_group.go

```diff
@@ -0,0 +1,169 @@
+package jupiterone
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/jupiterone/terraform-provider-jupiterone/jupiterone/internal/client"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ datasource.DataSource              = &userGroupDataSource{}
```

Comment:
I don't think this is necessary as line 24 will cause an error if `userGroupDataSource` doesn't implement `datasource.Datasource`

Same with the line right below this.  If this struct must implement those interfaces then there should be some other guarantee in the code itself rather than doing the blank identifier

Reply:
I thought this was the standard for compile time checks that I have seen in this repo, in tutorial, and via chatgpt, but you are the expert, so I am going to remove them.